README.md

```diff
@@ -37,6 +37,13 @@ git clone git@github.com:SparkPost/wordpress-sparkpost.git ~/src/wordpress/wp-co
 
 * Activate the plugin from admin panel
 
+## Running Tests
+* Follow the steps in [wp-cli](http://wp-cli.org/docs/plugin-unit-tests/)
```

Comment:
actually we need all of those. but i see your confusion (bcoz of heading). i will rename it to `setup and running tests`